app/src/main/java/org/covidwatch/android/ui/menu/MenuFragment.kt

```diff
@@ -63,4 +73,23 @@ class MenuFragment : Fragment(R.layout.fragment_menu) {
         findNavController().navigate(R.id.homeFragment)
     }
 
+    //Set red menu icon visible if maximum risk score > safe risk score
+    private fun updatePossibleExposureMenuItem(expSummary: CovidExposureSummary) {
+        val menuListAdapter: MenuAdapter = saveMenuList.adapter as MenuAdapter
+        var position = defaultPossibleExposuresPosition
+        //Find the "possible exposures" menu item
+        var size = menuListAdapter.getItemCount()
+        for (i in 0 until size) {
```

Comment:
It's cleaner just to return the items from the adapter and iterate them to find that item.